### PR TITLE
fixes

### LINKED ISSUE #497: Bug: Bulk delete functions - $bulk[channelID] returns error
### Bug Type

Function

### Bugged Function

```js
module.exports = {
    type: "messageDeleteBulk",
    channel: "$channelid",
    code: `
$channelSendMessage[$getGuildVar[log_msg_update];{newEmbed:{title:Bulk Message Delete}{color:Random}{timestamp}{field:Message Count:$get[c]}{field:Channel:<#$bulk[channelID]>}{field:IDs:\`\`\`$bulk[ids]\`\`\`}{footer:$username[$clientID]:$userAvatar[$clientID]}}]
$let[c;$getTextSplitLength]
$textSplit[$bulk[ids];,]
`
}
```

### Version of aoi.js

v6.5.0

### Version of Node.js

v18.17.1 (LTS)

### Excepted Output

When using `$bulk[channelID]` it should gimme channel ID but it returns error. i looked into the source code and the error i found was that, in the source code, it's made like `channelID : d.data.bulk.first().channelID` here it must be `channel.id` and not `channelID`. I hope this can be helpful. 

My exact Version of aoi - 6.5.6 (latest)

Note: If I am wrong in reporting this issue, please do enlighten me

### Relevant log output

```shell
It gives me some error like "cannot read properties of undefined (reading "deleteBrackets") In console.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### LINKED ISSUE #498: $execute[] function
### Featuring Type

Function

### So, what is it about?

Ability to trigger awaited commands directly without using `$ifAwaited[]`.

Usage: 
`$execute[...awaitedCmds]`

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### LINKED ISSUE #502: Bug: $fetchClientValues
### Bug Type

Function

### Bugged Function

```php
$fetchClientValues[guilds.cache.size]
```

Always returns the error about the function not being initialized, no matter what.

### Version of aoi.js

v6.5.0

### Version of Node.js

v16.11.0 (Lowest)

### Excepted Output

<Number>, amount of guilds.

### Relevant log output

```shell
AoiError: Invalid ClientShard Class is Not Initialised  
 { 
   Function : `$fetchClientValues`,
   Command : `eval`,
   Version : 6.5.x
 }
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### LINKED ISSUE #504: Bug: highest/lowestRole
### Bug Type

Function

### Bugged Function

```js
$userLowestRole / $userhighestRole returning some error when the user has no lowest role.
```

### Version of aoi.js

v6.5.0

### Version of Node.js

v16.11.0 (Lowest)

### Excepted Output

null or undefined

### Relevant log output

```shell
![img](https://cdn.discordapp.com/attachments/1058843428831629443/1179125896641269810/image.png?ex=6578a590&is=65663090&hm=80de847c3f0ac593cbf93cf43be43ebaac46edd45560b044b6c0a406718fa23d&)
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Faf4a**
Fixes:
$fetchClientValues, $bulk

Additions:
$awaitExecute[awaited], $resolveStickerID[resolver], $sendSticker[resolver]

Others:
Move custombox component to AoiError class.

Closes #497 
Closes #498 
Closes #502 
Closes #504

### CODE REVIEW

File: src/handler/AoiStart.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ module.exports = async (client, options) => {
         await require("./AoiLogs.js")(client);
     }
 
-    if (client.aoiOptions.aoiWarning !== false) {
+    if (client.aoiOptions.aoiWarning !== false && client.aoiOptions.aoiAutoUpdate === false) {
```
Comment (Leref) [inline src/handler/AoiStart.js]: This isn't true, since aoiAutoUpdate is always false, it has it's own thing.

File: src/classes/Database.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ class Database {
         this.tables = config.dataConfig.tables;
         this.type = moduleType;
         this.db.on(aoidb.DatabaseEvents.Connect, () => {
-            console.log(`[@akarui/aoi.db] Connected ${type} database`);
+            // console.log(`[@akarui/aoi.db] Connected ${type} database`);
```
Comment (Leref) [inline src/classes/Database.js]: We should use customBox logging

**Review by Leref — CHANGES_REQUESTED** [review]
some things

File: src/classes/Database.js (hunk 2 above)
Comment (Faf4a) [reply src/classes/Database.js:31]: Resolved.

File: src/handler/AoiStart.js (hunk 1 above)
Comment (Faf4a) [reply src/handler/AoiStart.js:8]: Reverted.

**Review by Leref — APPROVED** [review]

**Pull request merged**